examples/plot_dmp_2d_gain.py

```diff
@@ -0,0 +1,60 @@
+"""
+==================
+Effect of DMP Gain
+==================
+
+Demonstrates how modifying DMP gains (alpha_y, beta_y) affects 
+the resulting trajectory reproduction.
+
```

Comment:
empty lines should be removed

Reply:
Addressed in [f490348](https://github.com/dfki-ric/movement_primitives/pull/59/commits/f4903481e2804a15fb624d28cd8faa764906c5bf)